sdk/communication/azure-communication-administration/azure/communication/administration/aio/_phone_number_administration_client_async.py

```diff
@@ -70,7 +69,7 @@ def __init__(
         self._endpoint = endpoint
         self._phone_number_administration_client = PhoneNumberAdministrationClientGen(
             self._endpoint,
-            authentication_policy=HMACCredentialsPolicy(endpoint, credential),
```

Comment:
wondering if we really need the isAsync flag here?

Reply:
yes, because the get_authentication_policy function receives that optional parameter and passes it to the HMACCredentialPolicy object. Like this:

`return HMACCredentialsPolicy(endpoint, credential, decode_url=is_async)`